python/mathsys/bin/unix/x86/64/compile.sh

```
@@ -1,17 +1,10 @@
-nasm -f elf64 all.asm -o all.o
-
-# COMPILE -> RETURN
-cd ..
-cd ..
-cd ..
-cd ..
-cd ..
-cd ..
\ No newline at end of file
+(
+    cd python/mathsys/bin/unix/x86/64
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)